ROADMAP.md

```diff
@@ -8,28 +8,26 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.10
+## Rook Ceph 1.11
 
-The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+The following high level features are targeted for Rook v1.11 (January 2023). For more detailed project tracking see the [v1.11 board](https://github.com/rook/rook/projects/27).
 
-* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
-* Add command to the [krew plugin](https://github.com/rook/kubectl-rook-ceph) to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
-* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
 * Use more specific cephx accounts to better differentiate the source of Ceph configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Enable restoring a cluster from the OSDs after all mons are lost [#7049](https://github.com/rook/rook/issues/7049)
 * Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Support RBD mirroring across clusters with overlapping networks [#11070](https://github.com/rook/rook/issues/11070)
```

Comment:
Is this also applicable for CephFS mirroring?

Reply:
Yes, the same would apply to cephfs mirroring, so it's just a scenario priority to say we first want to make sure rbd mirroring is handled.